pkg/command/executor.go

```diff
@@ -15,8 +15,9 @@ type Executor interface {
 	Exec(ctx context.Context, name string, args ...string) error
 }
 
-// NewExecutor creates a new Executor instance.
-func NewExecutor(outW, errW io.Writer, inR io.Reader, cwd string, verbose bool, log *log.Logger) Executor {
+func NewExecutor(
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewExecutor should have comment or be unexported